packages/protocol/scripts/truffle/deploy_release_contracts.ts

```diff
@@ -0,0 +1,105 @@
+      'ReleaseGoldMultiSig'
+    )
+    const ReleaseGoldMultiSigProxy: ReleaseGoldMultiSigProxyContract = artifacts.require(
+      'ReleaseGoldMultiSigProxy'
+    )
+    const ReleaseGold: ReleaseGoldContract = artifacts.require('ReleaseGold')
+    const ReleaseGoldProxy: ReleaseGoldProxyContract = artifacts.require('ReleaseGoldProxy')
+    const releases = []
+    const handleJSONFile = async (err, data) => {
+      if (err) {
+        throw err
+      }
+      for (const releaseGoldConfig of JSON.parse(data)) {
+        const releaseGoldInitializeArgs = [
+          new Date(releaseGoldConfig.releaseStartTime).getTime() / 1000,
+          releaseGoldConfig.releaseCliffTime,
+          releaseGoldConfig.numReleasePeriods,
+          releaseGoldConfig.releasePeriod,
+          releaseGoldConfig.amountReleasedPerPeriod,
+          releaseGoldConfig.revocable,
+          releaseGoldConfig.beneficiary,
+          releaseGoldConfig.releaseOwner,
+          releaseGoldConfig.refundAddress,
+          releaseGoldConfig.subjectToLiquidityProvision,
+          releaseGoldConfig.initialDistributionRatio,
+          releaseGoldConfig.canValidate,
+          releaseGoldConfig.canVote,
+          registry.address,
+        ]
+        const releaseGoldMultiSigInitializeArgs = [
+          [releaseGoldConfig.releaseOwner, releaseGoldConfig.beneficiary],
+          2,
+          2,
+        ]
+        const initializeProxyContract = async (contractInstance, proxyInstance, args) => {
+          const initializeAbi = (contractInstance as any).abi.find(
+            (abi: any) => abi.type === 'function' && abi.name === 'initialize'
+          )
+          const callData = web3.eth.abi.encodeFunctionCall(initializeAbi, args)
+          const reciept = await proxyInstance._setAndInitializeImplementation(
+            contractInstance.address,
+            callData as any
+          )
+          return reciept.tx
+        }
+        const releaseGoldProxyInstance = await ReleaseGoldProxy.new()
+        const releaseGoldInstance = await ReleaseGold.new()
+        const releaseGoldMultiSigProxyInstance = await ReleaseGoldMultiSigProxy.new()
+        const releaseGoldMultiSigInstance = await ReleaseGoldMultiSig.new()
+        await goldToken.transfer(
+          releaseGoldProxyInstance.address,
+          releaseGoldConfig.amountReleasedPerPeriod * releaseGoldConfig.numReleasePeriods,
+          { from: releaseGoldConfig.releaseOwner }
+        )
+        const releaseTxHash = await initializeProxyContract(
+          releaseGoldMultiSigInstance,
+          releaseGoldMultiSigProxyInstance,
+          releaseGoldMultiSigInitializeArgs
+        )
+        const multiSigTxHash = await initializeProxyContract(
+          releaseGoldInstance,
+          releaseGoldProxyInstance,
+          releaseGoldInitializeArgs
+        )
+        await releaseGoldInstance.transferOwnership(releaseGoldMultiSigProxyInstance.address)
```

Comment:
This isn't doing what it should be doing because it's not being called via the proxy